app/docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: "3.6"
+
+services:
+  backend:
+    image: shiksha-lms-service
+    container_name: "shiksha-lms-service"
+    restart: always
+    ports:
+      - 5000:5000
+    env_file:
+      - /home/ubuntu-shiksha-lms-service/.env
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)